src/gridcoin/upgrade.cpp

```diff
@@ -95,10 +95,10 @@ bool Upgrade::CheckForLatestUpdate(bool ui_dialog, std::string client_message_ou
         return false;
     }
 
-    if (GithubReleaseTypeData.find("leisure"))
+    if (GithubReleaseTypeData.find("leisure") != std::string::npos)
```

Comment:
This check also suffers from a case-sensitive check. One possible solution:

```suggestion
    boost::to_lower(GithubReleaseTypeData);
    
    if (GithubReleaseTypeData.find("leisure") != std::string::npos)
```

...which will need:
 
```c++
#include <boost/algorithm/string/case_conv.hpp>
```

Reply:
Done =)